src/libostree/libostree.sym

```diff
@@ -373,12 +373,10 @@ global:
  *                         NOTE NOTE NOTE
  */
 
-/* Remove comment when first new symbol is added
-LIBOSTREE_2016.XX {
+LIBOSTREE_2017.3 {
```

Comment:
Right. It definitely makes upgrading easier. One thing about the symbol versions is that they also get used in the debian symbols file - https://github.com/endlessm/ostree/blob/debian-master/debian/libostree-1-1.symbols. Since that wasn't updated, nothing will happen (except for [dpkg-gensymbols complaining about it](https://obs-master.endlessm-sf.com/package/live_build_log?arch=x86_64&package=ostree&project=eos%3Amaster%3Aendless&repository=endless)), but in theory it would be nice if it was so that programs using libostree get a proper versioned dependency if they use this new symbol.

Meh. I guess it's fine the way it is.